geps/gep-713/index.md

```diff
@@ -1,1157 +1,1094 @@
+  - group: gateway.networking.k8s.io/v1
+    kind: Gateway
+    name: my-gateway
+  rules:      ## the "spec proper" describing the intent, i.e. color the traffic blue
+    color: blue
+```
+
+#### Properties of metaresources
+
+##### Targetability (“where”)
+
+Metaresources specify one or more target resources or specific sections of resources, whose behavior the metaresource intends to augment.
+
+##### Semantics (“why”)
+
+Targeting a resource (or section of a resource) must be interpreted within a given semantics that is proper to the metaresource kind.
+
+Two different metaresource kinds that allow targeting resources of the same given kind X may have very different semantics, not only because the purpose of the two metaresource kinds differ, but also because the mechanics of calculating and applying the augmented behavior differ.
+
+Often, the semantics of a metaresource is tightly coupled to the relationships and connections a target has with other kinds of objects, typically organized in a hierarchy of nested contexts. In this sense, targeting a given resource kind may have the semantics of spanning effect across yet other objects to which the target is related.
+
+##### Mergeability (“how”)
+
+Metaresources define so-called *merge strategies* that dictate how multiple instances of the metaresource affecting the same resource (or section of a resource) should be handled.
+
+The merge strategies typically include strategies for dealing with conflicting and/or missing specs, such as for applying default and/or override values on the target resources.
+
+### Policies and Policy Attachment
+
+Often when applying a metaresource, the intent has to do with specifying *rules* that control the behavior of the targets in action. E.g., for networking resources, the augmented behavior specified by a metaresource targeting an object can relate to conditions to let traffic flow or to block traffic at particular points of the network; in a different scenario, it can describe rules to mutate the traffic while it flows through the network at the point represented by the target object.
+
+The semantics of *applying rules to a target* characterizes these particular instances of metaresources as *policies*, thus here emerging in relation to this pattern the concept of *Policy Attachment*.
+
+Policies are therefore a strict subset of metaresources which distinguish itself subtly from the whole on the fact that the augmented behavior described by policies resemble *conditional rules*, while the mechanism of targeting an object or set of objects (“attaching to objects”) is nonetheless exactly the same as in the general definition of metaresources.
+
+Without loss of generality, the very activation of the context (targets) of metaresources can be understood as the _conditions_ for the enforcement of the intended augment behavior (rules) that are specified in the metaresources, and therefore these can often be refered to as policies interchangeably.
```

Comment:
typo in refered.  Also, in the phrase "these can often be refered...", what is "these"?  The targets or rules or something else?

```suggestion
Without loss of generality, the very activation of the context (targets) of metaresources can be understood as the _conditions_ for the enforcement of the intended augment behavior (rules) that are specified in the metaresources, and therefore these can often be refered to as policies interchangeably.
```
```suggestion
Without loss of generality, the very activation of the context (targets) of metaresources can be understood as the _conditions_ for the enforcement of the intended augment behavior (rules) that are specified in the metaresources, and therefore these can often be referred to as policies interchangeably.
```

Reply:
"these" == "metaresources"